CHANGELOG.md

```diff
@@ -7,11 +7,22 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+### Changed
+- Made username and password fields required on login and registration pages. #3795
+
 ### Fixed
-- Ports in Hadoop exploiter configuration can no longer be floating-point numbers.
-- Ports in Log4Shell exploiter configuration can no longer be floating-point numbers.
-- Ports in MSSQL exploiter configuration can no longer be floating-point numbers.
+- Ports in Hadoop exploiter configuration can no longer be floating-point numbers. #3734
+- Ports in Log4Shell exploiter configuration can no longer be floating-point numbers. #3734
+- Ports in MSSQL exploiter configuration can no longer be floating-point numbers. #3734
+- SMB exploiter configuration bug that wouldn't allow submission. #3849
+- Hadoop exploiter configuration bug that wouldn't allow submission. #3850
 
+### Removed
+- Fingerprinter configuration from UI. #3769, #3826
```

Comment:
We're planning on adding this back in, though. Right?

Reply:
Yes, I figured we should leave it in for now and remove it when we actually add it back in.

Reply:
Ok.